Tasks/google-play-release/task.json

```diff
@@ -87,6 +94,15 @@
             "defaultValue": "",
             "required": false,
             "helpMarkDown": "Path to the file specifying the release notes for the APK you are publishing."
+        },
+        {
+            "name": "additionalApks",
+            "type": "multiLine",
+            "label": "Additional APK paths",
+            "defaultValue": "",
+            "groupName": "advanced",
+            "required": false,
+            "helpMarkDown": "Paths to additional APK files you want to publish to the specified track. Wildcards can be used. For example, _\\*\\*/\\*.apk_ to match the first APK file, in any directory. Supports multiple lines or minimatch patterns."
```

Comment:
Should we add a quick e.g.? Maybe something like `Paths to additional APK files you want to publish to the specified track (e.g. an x86 build)`?